### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by NayanKaran**
Add mobile menu to the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,10 +16,16 @@
   <body>
     <header>
       <div id="logo">Mr. Mandal</div>
+      <div id="overlay">
+        <img id="close-button" src="./images/icons/close-icon.svg" alt="close button">
+        <a  class="menu-link" href="#projects">Portoflio</a>
+        <a  class="menu-link" href="#about-myself">About</a>
+        <a class="menu-link" href="#contact-form">Contact</a>
+      </div>
```
Comment (Mre55) [inline index.html:23]: Kindly, fix the background to be `blur` and `font-weight` of the menu option when the menu options appear. You can use something like this `filter: blur(..px)` or you can use anything that can work for you to achieve the last result as shown here in the Figma design:

| Your Design | Figma Design |
|-------------|---------------|
| image | image

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @NayanKaran,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
**[REQUIRED]** Please change your Pull Request description to make it more short summary of what you did in this Pull Request. Currently it's more generic and don't give the full information of what you achieve.

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (NayanKaran) [reply index.html:23]: Thank you for your valuable feedback. I have made the required changes. @Mre55

**Review by codepantha — APPROVED** [review]
Hi @NayanKaran 👋, 

## STATUS: APPROVED ✔️ 

Weldone implementing all the required changes 👏 👏 
Your project is now complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### To Highlight 🎯 ⭐ 
- Meaningful commits ✔️ 
- Linter checks ✔️ 
- Beautiful UI ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**